docs/TRANSLATIONS.md

```diff
@@ -1,116 +1,5 @@
-```
-tools/python/strings_utils.py -l ar -pm
-```
-By default, it searches `strings.txt`, to check `types_strings.txt` add a `-t` option.
-There are many more other options, e.g. print various translation statistics, validate and re-format translation files.
-Check `tools/python/strings_utils.py -h` to see all of them.
-
-To check consistency of types_strings.txt with categories.txt run:
-```
-ruby tools/ruby/category_consistency/check_consistency.rb
-```
-
-## Automatic translations
-
-In some cases automatically translated strings are better than no translation at all.
-There are two scripts to automate given string's translation into multiple languages.
-Please [install Translate Shell](https://www.soimort.org/translate-shell/#installation) first to be able to run them.
-
-### DeepL + Google Translate fallback
-
-The first one uses free DeepL API where possible and provides a significantly better quality translations.
-It requires registering a [DeepL account](https://www.deepl.com/pro#developer) and [getting API key](https://www.deepl.com/account/summary).
-You may be asked for a credit card for verification, but it won't be charged.
-Requires Python version >= 3.7.
-
-```bash
-export DEEPL_FREE_API_KEY=<your DeepL API key here>
-# Generates translations in both categories.txt and strings.txt formats at the same time:
-tools/python/translate.py English text to translate here
-# Use two-letter language codes with a colon for a non-English source language:
-tools/python/translate.py de:German text to translate here
-```
-
-### Google Translate only
-
-The second one is not recommended, it uses Google API and sometimes translations are incorrect.
-Also it does not support European Portuguese (pt or pt-PT), and always generates Brazil Portuguese.
-
-```bash
-# Generates translations in categories.txt format
-tools/unix/translate_categories.sh "Route"
-# Translations in strings.txt format
-DELIM=" = " tools/unix/translate_categories.sh "Route"
-```
-
-## Technical details
-
-Most of the translation files (strings, types_strings...) are in Twine file format ([syntax reference](https://github.com/organicmaps/twine/blob/organicmaps/README.md)).
-OM uses a custom version of the [Twine](https://github.com/organicmaps/twine)
-tool (resides in `tools/twine/` submodule) to generate platform-native (Android, iOS)
-localization files from a single translation file.
-
-The `tools/unix/generate_localizations.sh` script launches this conversion
-(and installs Twine beforehand if necessary).
-
-Search keywords translation files use a custom format described in the beginning of `data/categories.txt`.
-
-A `tools/python/clean_strings_txt.py` script is used to sync `strings.txt` with real UI strings usage as found in the codebase.
-
-There are preliminary plans to refactor translations workflow and migrate to Weblate.
+https://hosted.weblate.org/projects/organicmaps/
```

Comment:
We need a lot more detail than just this. Someone should probably instruct a translation team member on how to contribute translations (just a few sentences) and also instruct a developer on how to add new strings to the app (or invalidate / request changes to existing strings!) in a way that works with their local C++/Android/Xcode/Git workflow as well as during the PR approval process. (Do they need to use Google Translate to pre-fill all the missing languages? Just edit the XML file for English? What about C++ strings that need to be copied to both iOS and Android locale files? Do we just edit the XML manually by hand? Where are these files all at? etc)

Reply:
Developers no longer need to worry about translations. Just develop as usual in Android Studio and Xcode, adding English strings. Translators will handle the rest. No more pain!

Reply:
> Google Translate

Just in case, if you want to use machine translation, try DeepL first if your language is supported. Not only it often offers superior translation quality, but also the workflow is much better (e.g. it allows you to select alternative translations on a word-by-word basis, automatically rewords the rest of the sentence accordingly) and also it has an API allowing to use it in scripts even in the free tier.

Reply:
I've added more details to TRANSLATION.md CC: @zyphlar

Reply:
> Just in case, if you want to use machine translation, try DeepL first if your language is supported. Not only it often offers superior translation quality, but also the workflow is much better (e.g. it allows you to select alternative translations on a word-by-word basis, automatically rewords the rest of the sentence accordingly) and also it has an API allowing to use it in scripts even in the free tier.

We will definitely try DeepL in Weblate as the next step. Currently, it is set up to automatically add translations using the translation memory, which helps propagate translations between components, such as from Android to iOS and vice versa.

Reply:
Updated TRANSLATIONS.md. Deepl will be configured on Weblate.